diff --git a/node_modules/expo-image/ios/ContentPosition.swift b/node_modules/expo-image/ios/ContentPosition.swift
index 73c34ca..905f068 100644
--- a/node_modules/expo-image/ios/ContentPosition.swift
+++ b/node_modules/expo-image/ios/ContentPosition.swift
@@ -10,7 +10,7 @@ typealias ContentPositionValue = Either<Double, String>
  - Note: Its intention is to behave like the CSS [`object-position`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position) property.
  */
 struct ContentPosition: Record {
-  static let center = Self()
+  nonisolated(unsafe) static let center = Self()
 
   @Field
   var top: ContentPositionValue?
